docs/conf.py

```diff
@@ -26,7 +26,16 @@
 # The short X.Y version
 version = "0.25"
 # The full version, including alpha/beta/rc tags
-release = "0.25.0"
+release = "0.45.0"
```

Comment:
@mtreinish bad merge conflict, right?

```suggestion
release = "0.25.0"
```

Reply:
It's actually 0.44.0 now and will be 0.44.1 on the 0.25.1 release, but I had fixed this in #10644 so I didn't view it as critical for this PR.